### DISCUSSION TITLE
Welcome to Gemini Discussions!

### DISCUSSION THREAD

**Discussion by Gabe-H** [Gabe-H/Gemini#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by huihuang0513** [comment]
双子座Gemini 请问在何时升级完毕？什么时候钱包能正常打开？Gembank是否会同钱包一起上线？

**Reply by Gabe-H** [reply]
Sorry, this app works in conjunction with Spotify, and isn't related to the cryptocurrency website Gemini.

**Reply by heirenxiongdi** [reply]
Hello, I'd like to ask gembank, GemEx, how about ecological improvement thank u
